release/pkg/operations/upload.go

```diff
@@ -80,11 +80,12 @@ func UploadArtifacts(r *releasetypes.ReleaseConfig, eksArtifacts map[string][]re
 				}
 			}
 
+			// If the artifact is a helm chart, skip the skopeo copy as it's handled separately.
+			if strings.HasSuffix(artifact.Image.AssetName, "helm") {
+				continue
+			}
+
 			if artifact.Image != nil {
-				// If the artifact is a helm chart, skip the skopeo copy as it's handled separately.
-				if strings.HasSuffix(artifact.Image.AssetName, "helm") {
-					continue
-				}
```

Comment:
I'm struggling to see how this makes things better. Just the opposite it looks like.